src/custom.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '*.svg' {
+  import React = require('react')
+
+  export const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>
+  const src: string
+  export default src
+}
```

Comment:
Remove it. We don't use svgs here this way. Take a look at the folder components/ChevronDown and do the same